tests/phpunit/tests/multisite/getMainSiteId.php

```diff
@@ -88,6 +88,17 @@ public function test_get_main_site_id_returns_main_site_in_switched_context() {
 			$this->assertSame( $main_site_id, $result );
 		}
 
+		/**
+		 * @ticket 55802
+		 */
+		public function test_get_main_site_id_with_different_network_cache_id() {
```

Comment:
Are you able to add messages to each of these so it's clear which assertion is causing a test failure.

Reply:
Added